Containerfile

```diff
@@ -7,10 +7,17 @@ RUN microdnf -y install \
     --nodocs \
     golang \
     git-core \
+    nodejs \
+    nodejs-npm \
     python3 \
     python3-pip \
     && microdnf clean all
 
+RUN npm install -g corepack@~0.19 && \
```

Comment:
I had another thought after our pair programming. How about we use a package.json + package-lock.json anyway even if we don't prefetch with cachi2?

That could:
- enable dependabot updates with additional configuration
- improve reproducibility
- improve security thanks to hash checking (and thanks to dependabot CVE updates)

But we would have to make sure the lockfile gets respected - I think that means no `-g` for the npm install. We could instead do something like symlink the `node_module/.bin/corepack` binary into $PATH

Reply:
Yeah, this sounds like a good idea :+1: